app/assets/stylesheets/common/base/edit-category.scss

```diff
@@ -123,7 +123,7 @@ div.edit-category {
 
     .select-kit {
       &.tag-chooser {
-        width: $category-settings-width;
```

Comment:
any mobile concerns here?

Reply:
No seems fine, I changed this because on desktop it was way too cramped and it squashed the text

![image](https://user-images.githubusercontent.com/920448/114964938-0549f680-9eb3-11eb-82e7-f35e46bb0ca5.png)